dev/ToastNotifications/ToastActivationCallback.h

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "NotificationActivationCallback.h"
+
+struct ToastActivationCallback : winrt::implements<ToastActivationCallback, INotificationActivationCallback>
+{
+    HRESULT __stdcall Activate(
+        LPCWSTR appUserModelId,
+        LPCWSTR invokedArgs,
+        [[maybe_unused]] NOTIFICATION_USER_INPUT_DATA const* data,
+        [[maybe_unused]] ULONG dataCount);
+};
+
+struct ToastActivationCallback_factory : winrt::implements<ToastActivationCallback_factory, IClassFactory>
```

Comment:
Not really. Will keep this comment open to see if I could find some guidelines for it.

Reply:
Since PushNotificationBackgroundTaskFactory is the previous usage, let's follow it same style.
```suggestion
struct ToastActivationCallbackFactory : winrt::implements<ToastActivationCallbackFactory, IClassFactory>
```